src/tests/commonTests.ts

```diff
@@ -20,10 +20,20 @@ type TagAndPage = {
 
 export const HYDRATED_ATTR = config.hydratedFlag.name;
 
+/**
+ *  Helper for determining if a string is a tag or HTML
+ * @param {string} tagOrHTML - string that will be parsed
+ * @returns {boolean} true if tagOrHTML is a tag or html
+ */
 function isHTML(tagOrHTML: string): boolean {
```

Comment:
I think we can drop the doc on anything that's not exported since these are module-internal utils.